src/c++/perf_analyzer/test_utils.h

```diff
@@ -26,13 +26,55 @@
 
 #pragma once
 
+#include <atomic>
 #include <cmath>
 #include <cstdint>
 #include <numeric>
+#include <thread>
 #include <vector>
 
 namespace triton { namespace perfanalyzer {
 
+// FIXME
+class WatchDog {
```

Comment:
Can you provide some literature for what a watch dog concept is (in this context)? I'm unfamiliar and google seems to provide a different description than how it's used here.

Reply:
done